php-fpm/files/symfony.ini

```diff
@@ -0,0 +1,11 @@
+date.timezone = Europe/Paris
+xdebug.max_nesting_level = 1000
+xdebug.remote_enable = 1
+xdebug.remote_connect_back = 1
+xdebug.profiler_output_dir = "/var/logs/xdebug"
+xdebug.idekey = PHPSTORM
+memory_limit = 512M
+display_errors = on
+realpath_cache_size=4096k
+realpath_cache_ttl=600
+opcache.max_accelerated_files=20000
```

Comment:
Blank line at the end ! Use the editorconfig extension for your editor, there is a .editorconfig are the root of the project which take care for you to this :wink: